ui/components/component-library/text/text.js

```diff
@@ -29,61 +29,72 @@ export const ValidTags = [
   'strong',
   'ul',
   'label',
+  'input',
```

Comment:
We are using the Text component and polymorphic prop to create the input element in TextInputBase component PR https://github.com/MetaMask/metamask-extension/pull/16043/files#diff-d3b7305f6fb5c8d3d68007b90b8edefae588ed1141b5319e3785c4bc4f03c218R138-R167